README.md

```diff
@@ -219,6 +219,19 @@ To use Webpacker with [Elm](http://elm-lang.org), create a new app with `rails n
 The Elm library and core packages will be added via Yarn and Elm itself. An example `Main.elm` app
 is also added to your project in `app/javascript` so that you can experiment with Elm right away.
 
+## Caching Manifest
+
+Webpacker caches manifest file in production,
+however in the development and test environments where
+your code is most likely to change on each run, manifest is reloaded
+to ensure that latest change and packs has been picked up.
+If you want to toggle this behavior in certain
+environments you can do it in your `config/webpack/configuration.yml`
```

Comment:
I don't think "cache" is the best word to describe this behavior. Perhaps invert it with some form or "reload"? Also, should we apply this behavior more broadly to all file loaders and name it as such? Seems like we'd want `configuration.yml` to reload in dev mode too.